### PR TITLE
Add IT_logos library

### PR DESCRIPTION
**Pull request by pclainchard**
Hi, 
It is my first PR for an OpenSource project, so I hope I followed the guidelines correctly :smile: 

I have a lot of fun creating these logos, and if needed, I can add more of them or integrate some that have been made by someone else.

Cheers, 
Pierre

### CODE REVIEW

File: libraries.json [hunk 1]
```diff
@@ -124,5 +124,14 @@
     "source": "h7y/dropdowns.excalidrawlib",
     "preview": "h7y/dropdowns.png",
     "date": "2021-01-04"
+  },
+  {
+    "name": "IT Logos",
+    "description": "Collection of IT logos, including : SoapUI, SmartBear, GitKraken, BonitaSoft, Docker, Twitter, Strava, VueJS, Slack, SalesForce, Angular, React, LinkedIn, GitLab and Kanoma_IT",
```
Comment (lipis) [inline libraries.json:130]:
```suggestion
    "description": "Collection of IT logos, including: SoapUI, SmartBear, GitKraken, BonitaSoft, Docker, Twitter, Strava, VueJS, Slack, SalesForce, Angular, React, LinkedIn, GitLab and Kanoma_IT",
```

File: libraries.json [hunk 2]
```diff
@@ -124,5 +124,14 @@
...
+    "author": "Pierre Clainchard",
+    "url": "https://github.com/pclainchard",
+    "source": "pclainchard/few_IT_logos.excalidrawlib",
+    "preview": "pclainchard/few_IT_logos.png",
```
Comment (lipis) [inline libraries.json:134]:
```suggestion
    "source": "pclainchard/it-logos.excalidrawlib",
    "preview": "pclainchard/it-logos.png",
```

Comment (lipis) [reply libraries.json:134]: Can you also rename the actual files?

**Comment by lipis** [comment]
Awesome.. a few comments.

**Comment by pclainchard** [comment]
I've just updated the file names :)

**Review by lipis — APPROVED** [review]

**Pull request merged**